### ISSUE TITLE
New icon is broken when rendered in small sizes (<34px) in low-DPI screens (macOS)

### ISSUE THREAD

**Issue by danilofuchs** [microsoft/vscode#120935]
- VS Code Version: 1.55.1 (latest stable)
- OS Version: macOS 11.2.3
- Monitor resolution: 1080p

Steps to Reproduce:

1. Open VSCode
2. Go to Activity Monitor
3. Search for 'Code'
![image](https://user-images.githubusercontent.com/17657014/114195807-2c3e8080-9927-11eb-966f-ae72b8bdd1af.png)

A related issue on alt-tab-macos: https://github.com/lwouis/alt-tab-macos/issues/901

Icons above 34px are fine, below that are all broken.

Apparently, only happens on low-DPI displays (1080p)

Does this issue occur when all extensions are disabled?: Yes

**Comment by jamesdh** [comment]
This also effects alternative task-switchers, such as [Contexts](https://contexts.co):
![Screen Shot 2021-04-09 at 9 52 12 AM](https://user-images.githubusercontent.com/1085322/114198762-4d987000-9919-11eb-834b-bd2f19cf7722.png)

**Comment by tryangul** [comment]
Having the same issue on Catalina 10.15.7—in both activity monitor and contexts.

**Comment by gjsjohnmurray** [comment]
1.55.1 was supposed to have fixed this.

See https://github.com/microsoft/vscode/issues/120305#issuecomment-813180310

Does a macOS restart make any difference? Or the `killall Dock` command @misolori prescribed in the post linked above?

**Comment by danilofuchs** [comment]
I restarted my Mac and ran `killall Dock` and it did not solve it

**Comment by misolori** [comment]
I am not able to reproduce and have the same version as you (VS Code 1.55.1 on macOS 11.2.3) as mentioned before [this was fixed in the 1.55.1 release](https://github.com/microsoft/vscode/issues/120305):

Here is 1.55.0
![image](https://user-images.githubusercontent.com/35271042/114220414-d4048000-9920-11eb-91cb-1850229ba569.png)

And here it is after I update to 1.55.1
![image](https://user-images.githubusercontent.com/35271042/114220485-eb436d80-9920-11eb-8e9b-7b548bb34180.png)

## Clearing your icon cache
You might have an icon cache problem. Can you try:

1. Ensure you are on the latest version 1.55.1

2. Restart your dock via `killall Dock` in the terminal

3. Run these scripts in your terminal (which updates the timestamp of the icons):
```
sudo touch /Applications/Visual\ Studio\ Code\ -\ Insiders.app
sudo touch /Applications/Visual\ Studio\ Code\ -\ Insiders.app/Contents/Info.plist;
sudo touch /Applications/Visual\ Studio\ Code\ -\ Insiders.app/Contents/Resources/Code.icns;
killall Dock Finder
```

4. Force the IconService cache to be rebuilt [using these steps](https://mjtsai.com/blog/2019/02/01/clearing-the-icon-services-cache-in-mojave/)

**Comment by danilofuchs** [comment]
After another system reboot, the icons were perfect once again. I will close the issue as it indeed seems to be resolved as of 1.55.1/1.55.2
